from_cpython/Modules/itertoolsmodule.c

```diff
@@ -4080,6 +4080,30 @@ static PyMethodDef module_methods[] = {
     {NULL,              NULL}           /* sentinel */
 };
 
+PyTypeObject* Itertool_SafeDealloc_Types[] = {
+    // &combinations_type,
+    // &cwr_type,
```

Comment:
I copied the static array from `PyTypeObject *typelist[]` that has all the types in the file, then commented out those that I wasn't sure was safe yeah.